src/app/root/services/api/api.service.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { TestBed, async, inject } from '@angular/core/testing';
+import { Http } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+
+import { ApiService } from './api.service';
+
+let mockHttp;
+let spyHttpGet;
+let apiService: ApiService;
+
+describe('the api service', () => {
```

Comment:
Let's use `ApiService` in the describe so that the IDE can refactor it if the name should ever change